src/Twig/AssetRewritingVisitor.php

```diff
@@ -23,15 +23,15 @@ public function getPriority()
     /**
      * {@inheritdoc}
      */
-    protected function doEnterNode(\Twig_Node $node, \Twig_Environment $env)
+    protected function doEnterNode(\Twig_Node $node, \Twig\Environment $env)
```

Comment:
Can we switch this typehint to `Twig\Node\Node` instead of `Twig\Node`?

Reply:
Also: please add `use` statements for these classes instead of the fully qualified class name.